src/js/dismiss.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * DISMISS
+ *
+ * @package uri-component-library
+ */
+
+( function() {
+
+	'use strict';
+
+	window.addEventListener( 'load', init, false );
+
+	function init() {
+
+		var els, i;
+
+		els = document.querySelectorAll( '.cl-countdown, .cl-notice' );
+
+		for ( i = 0; i < els.length; i++ ) {
+			setup( els[i] );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['setup' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/uriweb/uri-component-library/pullRequest?prid=4536390)